debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/ColumnDefinitionParserListener.java

```diff
@@ -37,6 +38,8 @@
     private final DataTypeResolver dataTypeResolver;
     private final TableEditor tableEditor;
     private ColumnEditor columnEditor;
+    private boolean uniqueColumn;
+    private Optional<Boolean> optionalColumn;
```

Comment:
There's not much point in using `Optional` as a field. Simply use `Boolean`.

Reply:
@gunnarmorling I've initially used the `Boolean` but I belive that `Optional` expess three state semantics not-set/false/true better than `null`/false/true.
`Optional` explicitly forces you to check, `null` checking is only optional

Reply:
I dare to disagree :) Keep in mind `Optional` was only foreseen as a method return type, e.g. see what Stuart Marks has to say on the issue: https://stackoverflow.com/questions/23454952/uses-for-optional

Reply:
@gunnarmorling ok, you are the boss ;-) - changed to `Boolean`.

BTW let me try (https://www.logicallyfallacious.com/tools/lp/Bo/LogicalFallacies/21/Appeal-to-Authority) as counterargument and point you to https://twitter.com/mariofusco/status/518729164384645120 :-)

And a chapter in https://livebook.manning.com/#!/book/modern-java-in-action/chapter-11/1

Reply:
LOL, who's that Fusco guy ;)